src/app/rpm-ostree-countme.timer

```diff
@@ -8,6 +8,7 @@ ConditionPathExists=/run/ostree-booted
 OnCalendar=weekly UTC
 AccuracySec=1u
 RandomizedDelaySec=1w
+Persistent=yes
```

Comment:
Should we add an `OnBootSec=` of e.g. 5 minutes or so? That way, (1) there's less concern that networking might not be up yet, and (2) given that this isn't necessary for system bring up, we spread the load across the time dimension.

Reply:
Good idea

Reply:
Reading https://www.freedesktop.org/software/systemd/man/systemd.timer.html#OnActiveSec= again, as we set `AccuracySec=1u`, this means that the timer will trigger systematically on each boot at 5 min precisely so I'm not sure that's a good idea.

I'm not sure we can do better than 'just' `Persistent=yes`.

Reply:
Hmm you're right, there doesn't seem to be a way to say "if we skipped a trigger, don't trigger until at least 5mins after boot".

So we'll just have to live with possibly failing in the case the machine is brought up after having slept for a while and networking isn't up yet.